Modules/Filtering/Thresholding/include/itkThresholdImageFilter.h

```diff
@@ -103,15 +103,15 @@ class ITK_TEMPLATE_EXPORT ThresholdImageFilter : public InPlaceImageFilter<TImag
   /** Get the "outside" pixel value. */
   itkGetConstMacro(OutsideValue, PixelType);
 
-  /** The values greater than or equal to the value are set to OutsideValue. */
+  /** The values greater than "thresh" are set to OutsideValue. */
   void
   ThresholdAbove(const PixelType & thresh);
```

Comment:
Why not rename this parameter to `threshold`?

Reply:
Outside scope of doc PR.